src/fides/ctl/core/export.py

```diff
@@ -174,45 +176,67 @@ def generate_system_records(
+        if system.privacy_declarations:
+            for declaration in system.privacy_declarations:
+                data_use = formatted_data_uses[declaration.data_use]
+                data_categories = declaration.data_categories or []
+                data_subjects = [
+                    formatted_data_subjects[data_subject_fides_key]
+                    for data_subject_fides_key in declaration.data_subjects
+                ]
+                dataset_references = declaration.dataset_references or [
+                    EMPTY_COLUMN_PLACEHOLDER
+                ]
+                cartesian_product_of_declaration = [
+                    (
+                        system.fides_key,
+                        system.name,
+                        system.description,
+                        system.data_responsibility_title,
+                        system.administrating_department,
+                        third_country_list,
+                        system_dependencies,
+                        declaration.name,
+                        category,
+                        data_use["name"],
+                        data_use["legal_basis"],
+                        data_use["special_category"],
+                        data_use["recipients"],
+                        data_use["legitimate_interest"],
+                        data_use["legitimate_interest_impact_assessment"],
+                        subject["name"],
+                        subject["rights_available"],
+                        subject["automated_decisions_or_profiling"],
+                        declaration.data_qualifier,
+                        data_protection_impact_assessment["is_required"],
+                        data_protection_impact_assessment["progress"],
+                        data_protection_impact_assessment["link"],
+                        dataset_reference,
+                    )
+                    for category in data_categories
+                    for subject in data_subjects
+                    for dataset_reference in dataset_references
+                ]
+                output_list += cartesian_product_of_declaration
+        else:
+            system_row = [
+                system.fides_key,
+                system.name,
+                system.description,
+                system.data_responsibility_title,
+                system.administrating_department,
+                third_country_list,
+                system_dependencies,
+                data_protection_impact_assessment["is_required"],
+                data_protection_impact_assessment["progress"],
+                data_protection_impact_assessment["link"],
             ]
-            output_list += cartesian_product_of_declaration
+            length_to_na = 12
+            start_position = 6
+            for i in range(length_to_na):
+                system_row.insert(i + start_position, EMPTY_COLUMN_PLACEHOLDER)
+            system_row.append(EMPTY_COLUMN_PLACEHOLDER)
```

Comment:
how come there is one extra append here?

Reply:
iirc there is one extra column that is added in the sequence here that would have `"N/A"` at the very end. Will see if I can add a comment or something in the docs block that cleans that up some but it was basically to preserve the column order with the data map as is